saleor/settings.py

```diff
@@ -528,6 +528,12 @@ def SENTRY_INIT(dsn: str, sentry_opts: dict):
     ],
 }
 
+# Max number entities that can be requested in single query by Apollo Federation
+# Federation protocol implements no securities on its own part - malicious actor
+# may build a query that requests for potentially few thousands of entities.
+# Set FEDERATED_QUERY_MAX_ENTITIES=0 in env to disable (not recommended)
+FEDERATED_QUERY_MAX_ENTITIES = int(os.environ.get("FEDERATED_QUERY_MAX_ENTITIES", 300))
```

Comment:
Maybe this should be configured to the same value as `RELAY_CONNECTION_MAX_LIMIT`? If we allow max 100 to be returned there, I think it's a reasonable value here as well.